card/mod/05_standard/set/all/links.rb

```diff
@@ -31,7 +31,11 @@ def web_link href, opts={}
   # link to a specific card
   def card_link name, opts={}
     text = (opts.delete(:text) || name).to_name.to_show @context_names
-    name = Card[name].cardname if Symbol===name
+    name = case name
+           when Symbol then Card[name].cardname
+           when Card   then name.cardname
+           else             name
+           end
```

Comment:
D'oh.

But, then skip_modules is still needed as mentioned.  I suppose it wouldn't be a bad thing to switch the default for [].  then Card[foo].include_sets (or what it is really called) is still nice and compact.